build/trend-scenarios.yml

```diff
@@ -82,6 +82,8 @@ parameters:
     arguments: --scenario mapaction $(jsonJobs) --property scenario=JsonMapAction --property protocol=http
   - displayName: Json Minimal APIs
     arguments: --scenario json $(minimalJobs) --property scenario=JsonMinimalApis --property protocol=http
+  - displayName: Json 400k
+    arguments: --scenario json_400k --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/json.benchmarks.yml --property scenario=Json400k
```

Comment:
You don't have to create specific scenarios if this is mostly for CI, just one and set the variable in this line for each value you want to test. More flexible.

Reply:
I don't understand this comment.

Reply:
`--scenario json_400k` means you created a scenario named `json_400k`. Now let's say you want to test with 1, 42, 1000, ... you would create a new scenario for each. It's nice if people need to chose from these, but I believe in our case this is just for CI, you can create a single scneario called "AllTheJsonsizes" (joking), and use the same scenario name, but change a variable which contains the size. In the end this file (azdo script) would look like:

```
- displayName: Json 400k
  arguments: --scenario AllTheJsonsizes --variable jsonSize=400000 --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/json.benchmarks.yml --property scenario=Json400k

- displayName: Json 1k
  arguments: --scenario AllTheJsonsizes --variable jsonSize=1000 --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/json.benchmarks.yml --property scenario=Json1K
...
```

So you don't have to create new scenarios. The same way we can vary the number of connections for Plaintext without having to create several scenarios.